packages/jest-cli/src/TestSequencer.js

```diff
@@ -59,6 +72,41 @@ export default class TestSequencer {
     return cache;
   }
 
+  _fileSizeRecurseDependencies(test: Test, sizes: Map<Path, number>): number {
+    const {resolver, hasteFS, config} = test.context;
+
+    const fileSize = (path): number => {
+      const cachedSize = sizes.get(path);
+      if (cachedSize != null) {
```

Comment:
```suggestion
      if (sizes.has(path)) {
```

Reply:
The problem with that is the Flow types - Flow will think the size can be null.

Reply:
>The problem with that is the Flow types - Flow will think the size can be null

Not sure why you need `cachedSize` at all, `sizes` is always defined and you should be good using @SimenB's code (plus removing line 79), no?